cypress/support/fragments/quickMarcEditor.js

```diff
@@ -1061,22 +1066,20 @@ export default {
     );
   },
 
-  checkHeaderFirstLine(
```

Comment:
please run test that use this method, to ensure that theese changes doesn't break anything

Reply:
This method was written but not used anywhere